static/assets/global.css

```
@@ -331,3 +331,94 @@ input[type='number'] {
+  width: 50rem;
+  left: -50rem;
+  right: -50rem;
+  top: -50rem;
+  bottom: -50rem;
+  border-radius: 50%;
+  margin: auto;
+  background: #9ba4b5;
+  z-index: -2;
+  pointer-events: none;
+  animation: rotate-then-fade 3.5s linear 1 forwards;
+  background: conic-gradient(
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)